awx/main/dispatch/worker/callback.py

```diff
@@ -0,0 +1,120 @@
+import logging
+import time
+import os
+import signal
+import traceback
+
+from django.conf import settings
+from django.db import DatabaseError, OperationalError, connection as django_connection
+from django.db.utils import InterfaceError, InternalError
+
+from awx.main.consumers import emit_channel_notification
+from awx.main.models import (JobEvent, AdHocCommandEvent, ProjectUpdateEvent,
+                             InventoryUpdateEvent, SystemJobEvent, UnifiedJob)
+
+from .base import BaseWorker
+
+logger = logging.getLogger('awx.main.dispatch')
+
+
+class CallbackBrokerWorker(BaseWorker):
```

Comment:
This is the worker implementation for callback events.  This is almost 100% copied from the callback receiver command code in 3.3.0; not much has changed here outside of the refactoring described above.